pkg/tests/tasks/traffic/egress/yaml_configs.go

```diff
@@ -15,7 +15,7 @@
 package egress
 
 var (
-	httpbinextServiceEntry = `
+	httpbinExternalServiceEntryHttpPortOnly = `
```

Comment:
Can you move this to `access_external_services_test.go`, since it's only used there?